model-optimizer/mo/utils/ir_reader/extenders/range_extender.py

```diff
@@ -0,0 +1,27 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from mo.utils.graph import Node
+from mo.utils.ir_reader.extender import Extender
+
+
+class RangeExtender(Extender):
+    op = 'Range'
+
+    @staticmethod
+    def extend(op: Node):
+        op['output_type'] = op['dtype']
```

Comment:
@achetver , please, take a look at this change.

Reply:
Why we cant't use `output_type` value for Range-4 operation restored from IR?

And what about Range-1? If we read and save IR with Range-1 operation it will be same (without `output_type` attribute)?

Reply:
I removed the extender since it is not needed for op which never was in IR.

Reply:
The v1::Range probably was in IR. For example, it may happen for the models created with --keep_shape_ops parameter if the Range was on a shape path which is const folded in nGraph. 

Also, the extender is still needed because v4 Range will be in IR and in order to correctly restore it we need to restore the data type. So the extender should look like this:

```
        if op.has_valid('output_type'):
            op['output_type'] = destination_type_to_np_data_type(op.output_type)
```